### ISSUE TITLE
Please support multiple binaries

### ISSUE THREAD

**Issue by blurayne** [marwanhawari/stew#59]
like intofuutils/tenv 

tf, tofu, atmos, tenv are all required.

**Comment by marwanhawari** [comment]
See also discussion here: https://github.com/marwanhawari/stew/issues/2

**Comment by marwanhawari** [comment]
Okay I actually think this is already supported in the v0.4.0 release.

Here is a case where there are N binaries in 1 asset (https://github.com/sxyazi/yazi):
![Image](https://github.com/user-attachments/assets/8560ab3b-66f6-4689-8def-503af77ee4d6)

Here is a case where there are N assets in 1 repo (https://github.com/SUPERCILEX/fuc):
![Image](https://github.com/user-attachments/assets/7dad0138-a799-4494-acfa-6b9881168bd4)

Tagging you @gdevenyi because you left [this comment](https://github.com/marwanhawari/stew/issues/2#issuecomment-2414562993) recently.

Could someone describe what isn't working and how to reproduce?

**Comment by marwanhawari** [comment]
Yeah I will update docs to make it clear that you can install multiple binaries from the same repo.

Originally, you actually couldn't install multiple binaries because there was a strict 1:1:1 connection between repo, asset, and binary. Once that strict connection between asset and binary was removed [in this PR](https://github.com/marwanhawari/stew/pull/31), you could start to install multiple binaries per repo/asset. That PR was included in the v0.4.0 release.
